stable/console-chart/templates/console-clusterrole.yaml

```diff
@@ -56,7 +58,10 @@ rules:
   apiGroups:
   - ''
   resources:
+  - configmaps
```

Comment:
To enable grabbing any needed configmaps, for example OCP console's configmap for their route